modules/service-order.nix

```diff
@@ -0,0 +1,61 @@
+# Ordering Services
+#
+# Given a set of services, make them run one at a time in a specific
+# order, on a timer.
+{ lib, ... }:
```

Comment:
Any reason to allow additional arguments here?

Since this is an attrset that isn't importable as a module, this doesn't really make sense to me as is. Was this initially started considering it could be implemented as a NixOS module? Could it be made one? Otherwise, would it make make more sense at a location like `../lib` or `./lib`?

Reply:
+1 that was sloppy of me

Reply:
Fixed up.